services/app-api/src/emailer/emailer.ts

```diff
@@ -180,181 +236,117 @@ function newSESEmailer(config: EmailConfiguration): Emailer {
                 statePrograms,
-                submitterEmails
+                dateAsked
             )
-            if (emailData instanceof Error) {
-                return emailData
+            if (localLogger) {
+                if (emailData instanceof Error) {
+                    console.error(emailData)
+                    return emailData
+                } else {
+                    localLogger(emailData)
+                }
             } else {
-                localEmailerLogger(emailData)
+                if (emailData instanceof Error) {
+                    return emailData
+                } else {
+                    return await this.sendEmail(emailData)
+                }
             }
         },
-        sendResubmittedStateEmail: async (
+        sendResubmittedStateEmail: async function (
             formData,
             updateInfo,
             submitterEmails,
             statePrograms
-        ) => {
+        ) {
             const emailData = await resubmitPackageStateEmail(
                 formData,
                 submitterEmails,
                 updateInfo,
                 config,
                 statePrograms
             )
-            if (emailData instanceof Error) {
-                return emailData
+            if (localLogger) {
+                if (emailData instanceof Error) {
+                    console.error(emailData)
+                    return emailData
+                } else {
+                    localLogger(emailData)
+                }
             } else {
-                localEmailerLogger(emailData)
+                if (emailData instanceof Error) {
+                    return emailData
+                } else {
+                    return await this.sendEmail(emailData)
+                }
             }
         },
-        sendResubmittedCMSEmail: async (
+        sendResubmittedCMSEmail: async function (
             formData,
             updateInfo,
             stateAnalystsEmails,
             statePrograms
-        ) => {
+        ) {
             const emailData = await resubmitPackageCMSEmail(
                 formData,
                 updateInfo,
                 config,
                 stateAnalystsEmails,
                 statePrograms
             )
-            if (emailData instanceof Error) {
-                return emailData
+            if (localLogger) {
+                if (emailData instanceof Error) {
+                    console.error(emailData)
+                    return emailData
+                } else {
+                    localLogger(emailData)
+                }
             } else {
-                localEmailerLogger(emailData)
+                if (emailData instanceof Error) {
+                    return emailData
+                } else {
+                    return await this.sendEmail(emailData)
+                }
             }
         },
     }
 }
 
+function newSESEmailer(config: EmailConfiguration): Emailer {
```

Comment:
I'm very glad you maintained these entry points. I think we can simplify things a bit more. Looking at it I think the main weirdness is that sendEmail is not like the other functions. And I forgot that we also duplicate this structure in our testing helper too. I think we could break out the sendTestEmail, sendSESEmail, sendLocalEmail functions into something that you pass into a single "emailer". I can sketch it out or we can meet and talk it through.